Makefile

```diff
@@ -1,3 +1,4 @@
 build  :; DAPP_BUILD_OPTIMIZE=0 DAPP_BUILD_OPTIMIZE_RUNS=0 dapp --use solc:0.6.12 build
 clean  :; dapp clean
 test   :; DAPP_BUILD_OPTIMIZE=0 DAPP_BUILD_OPTIMIZE_RUNS=0 dapp --use solc:0.6.12 test -v ${TEST_FLAGS}
+test-rpc :; dapp --use solc:0.6.12 test --rpc -m '_rpc_'
```

Comment:
Should use the same explicit envvars as the other targets so the same ESM bytecode is for sure used in all tests.

Reply:
OK

Reply:
done